driver.go

```diff
@@ -23,66 +23,69 @@ import (
 const (
 	driverName = "triton"
 	flagPrefix = driverName + "-"
-	envPrefix  = "TRITON_"
+	// SDC_ is for historical reasons
+	envPrefix  = "SDC_"
 )
 
 var (
-	defaultSdcAccount = ""
-	defaultSdcKeyFile = os.Getenv("HOME") + "/.ssh/id_rsa"
-	defaultSdcKeyId   = ""
-	defaultSdcUrl     = "https://us-east-1.api.joyent.com"
+	defaultTritonAccount = ""
+	defaultTritonKeyPath = os.Getenv("HOME") + "/.ssh/id_rsa"
+	defaultTritonKeyId   = ""
+	defaultTritonUrl     = "https://us-east-1.api.joyent.com"
 
 	// https://docs.joyent.com/public-cloud/instances/virtual-machines/images/linux/debian#debian-8
-	defaultSdcImage   = "debian-8"
-	defaultSdcPackage = "g3-standard-0.25-kvm"
+	defaultTritonImage   = "debian-8"
+	defaultTritonPackage = "g3-standard-0.25-kvm"
+
+	errUnimplemented = errors.New("UNIMPLEMENTED")
```

Comment:
@tianon you were suggesting I add this back in https://github.com/joyent/docker-machine-driver-triton/pull/1#issuecomment-213502583 ?